### DISCUSSION TITLE
How to run on Windows?

### DISCUSSION THREAD

**Discussion by XfedeX** [piqueserver/piqueserver#612] [Q&A]
Sorry if the question is dumb.
I installed with _PIP_ PiqueServer.
I execute command: _piqueserver_
and I get error: _[Errno 2] No such file or directory: 'C:\\Users\\Toshiba/.config/piqueserver\\config.json'_
Where do I can get the config files?
Why i can't just download a release and run it, like in Minecraft Servers?

**Comment by NotAFile** [comment]
Hi, please read the instructions for running piqueserver.

**Comment by feikname** [comment]
**Life Pro Tip**:  It's a good idea to always read a project's README file first before running it as those questions are usually answered there

As stated in piqueserver's README filme, which can be found [here](https://github.com/piqueserver/piqueserver/blob/master/README.rst) or also by scrolling down to below the file list in the [project's home page](https://github.com/piqueserver/piqueserver), like most other projects on GitHub:

\:rocket: Running
-----------------

Copy the default configuration as a base to work off

.. code:: bash

    piqueserver --copy-config

A-a-and lift off!

.. code:: bash

    piqueserver
    
    
 <hr>

 Perhaps piqueserver could copy a default config automatically if it detect's there's no current config file, as a quality of life improvement. However, if that happens it will be in a future release only, so, for now at least, you gotta follow these procedures.
 
 > Why i can't just download a release and run it, like in Minecraft Servers?
 
 Although possible, it's uncommon for Python-based software to be packed as an executable, they're usually installed using the `pip` [package manager](https://en.wikipedia.org/wiki/Package_manager), which gets packages from the [Python Package Index (PyPI)](https://pypi.org/). Minecraft servers on the other hand are based on Java technology, which packs all the programs dependencies into a single `.jar` file, which is actually just a `.zip` file.
 
 For user friendliness piqueserver could also be packed as a zip file for Windows distributions in the future. You can open a new feature request issue here asking for that, supposing there isn't one already open 😄.
